frontend/src/pages/contains-tracker-warning.module.scss

```
@@ -38,7 +92,8 @@
     flex-direction: column;
 
     .lead {
-      flex: 0.6;
+      // flex value is 0.35 to make sure the text is not too wide and to reflect design specs
+      flex: 0.35;
```

Since you're only setting a single property, might as well clarify which property that is (I had to look it up :P):

```suggestion
      flex-grow: 0.35;
```

---

This one is a bit tricky because doing `flex: 0.35` implicitly sets `flex-basis` and `flex-shrink` to `auto` or `0%` and `1` respectively. If you want to see exactly what is going on, I have gone ahead and updated it to show the following: 

`flex-basis: 0;
 flex-grow: 0.35;
      `

---

Ah OK, yeah in that case setting the other properties explicitly rather than implicitly sounds like a good idea :)